bindings/nodejs/lib/secret_manager/secret-manager.ts

```diff
@@ -113,14 +116,14 @@ export class SecretManager {
     }
 
     /**
-     * Signs a message with an Evm private key.
+     * Signs a message with an Secp256k1Ecdsa private key.
      */
-    async signEvm(
+    async signSecp256k1Ecdsa(
```

Comment:
It's been released now so it needs and entry in the changelog. You'll need to update your branch to have it I guess.